potypo/__main__.py

```diff
@@ -8,32 +8,27 @@
 from enchant import DictWithPWL, errors
 from enchant.checker import SpellChecker
 
-def main():
-    config = configparser.ConfigParser()
-    config.read('setup.cfg')
-    conf = config['potypo']
 
-    chunker_list = []
-    for chunker in conf['chunkers'].strip().split(","):
-        if "." in chunker:
-            components = chunker.rsplit('.',1)
+def load_classes(module, config):
```

Comment:
This method probably needs a better name, but I couldn't think of any.

Reply:
Works for me :)